client/jetpack-cloud/components/layout/nav.tsx

```diff
@@ -55,7 +55,7 @@ export default function LayoutNavigation( {
 			selectedText={
 				<span>
 					{ selectedText }
-					<Count count={ selectedCount } compact={ true } />
+					{ selectedCount !== undefined && <Count count={ selectedCount } compact={ true } /> }
```

Comment:
[Non-blocking, subjective preference]

Another way to write this might also be:

```suggestion
					{ Number.isInteger( selectedCount ) && <Count count={ selectedCount } compact={ true } /> }
```

Reply:
Oh -- we can also shorten up the declaration by doing

```jsx
<Count compact count={ selectedCount />
```

since adding a property without a value defaults it to `true`.